modules/miscutil/lib/inveniocfg.py

```diff
@@ -879,21 +885,25 @@ def cli_cmd_load_demo_records(conf):
                 "%s/bin/bibupload 1" % CFG_PREFIX,
                 "%s/bin/bibdocfile --textify --with-ocr --recid 97" % CFG_PREFIX,
                 "%s/bin/bibdocfile --textify --all" % CFG_PREFIX,
-                "%s/bin/bibindex -u admin" % CFG_PREFIX,
+                "%s/bin/bibindex -u admin -w collection" % CFG_PREFIX,
```

Comment:
If a change is required in index scheduling, then I guess we should update HOWTO Run and other documentation at the same time...